www.foia.gov/Makefile

```diff
@@ -0,0 +1,5 @@
+clean:
+	perl -pi -e 's,https://www.foia.gov/,,g' *html
```

Comment:
Shouldn't these urls be prefixed with `/` rather than a true relative path? For example, if you're on a nested path `/news/thing.html` wouldn't browsers interpret this as `/news/foia.css` when it should be `/foia.css`?